web/views.py

```
@@ -6772,3 +6786,69 @@ def delete_post(request, post_id):
     if request.method == "POST":
         post.delete()
     return redirect("social_media_dashboard")
+
+
+@login_required
+@require_POST
+def add_custom_badge(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `add_custom_badge`

(ANN201)